core/modules/macros/contrastcolour.js

```diff
@@ -29,6 +29,22 @@ exports.params = [
 Run the macro
 */
 exports.run = function(target,fallbackTarget,colourA,colourB) {
```

Comment:
But I see a reference to the current palette and the `tag-background` item? It's not good for a generic macro to have code that is specific to one of it's applications.